docs/tutorials/Biometric-Liveness-Verification/Readme.md

```diff
@@ -0,0 +1,364 @@
+# Biometric Liveness Verification
+
+<!-- AUTHOR romanstrobl 2024-09-26T00:00:00Z -->
+<!-- SIDEBAR _Sidebar.md sticky -->
+<!-- TEMPLATE tutorial -->
+
+This tutorial explains how Wultra's Biometric Liveness Verification components can be deployed, configured and integrated into your application. This solution enables to verify users using advanced biometric verification techniques against their static photos, such as passport photos.
+
+## Introduction
+
+Wultra's Biometric Liveness Verification solution consists of two components which require to be deployed. The components are available as Docker images in Wultra JFrog Artifactory.
+
```

Comment:
Some high-level explanation or picture would be great. To understand the whole concept.

Reply:
Ok, I will write an explanation.